app/Models/Event.ts

```diff
@@ -75,7 +75,7 @@ export default class Event extends BaseModel {
       ? this.reviews
           .map((review) => review.rating)
           .reduce((count, el) => {
-            return count + el
+            return parseInt(count.toString()) + parseInt(el.toString())
```

Comment:
```suggestion
            return parseInt(count.toString(), 10) + parseInt(el.toString(), 10)
```
add radix to parseInt function